Form: update Customize Item demo description (#2189)
DevExpress/devextreme-demos: JSDemos/Demos/Form/CustomizeItem/description.md

You can use the [items[]](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#items) array to configure all form items. This array can contain strings ([formData](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#formData) field names) and objects (item configurations). 

Use a string to create a simple item with default configuration as shown for the **Email** item. 

To change the default settings, declare an item configuration object. Use the [dataField](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#dataField) property to bind an item to a field in the [formData](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#formData) object. Use the [editorType](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#editorType) property to specify an item's data editor or configure the editor in the [editorOptions](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#editorOptions) object. You can also specify any other properties described in the [SimpleItem](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/) section.

> To customize item labels, use the [label](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/label/).[template](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/label/#template) property. The demo uses this property to add icons to the labels. Examine the **Additional Notes** item's implementation to learn how to add an icon with a tooltip to the label. 

Reviewer: ```suggestion
To customize item labels, use the [label](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/label/).[template](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/label/#template) property. The demo uses this property to add icons to the labels. Refer to the **Additional Notes** item's implementation for instructions on how to add an icon with a tooltip to the label. 
```
